docs/api.yaml

```diff
@@ -36,7 +36,19 @@ info:
 
     Here major and breaking changes to the API are listed by version.
 
-    ## ODK Central v2023.4
+    ### ODK Central v2023.5
```

Comment:
Should this just be `##`? Can you put backticks around the appropriate words below?